IBPSA/Resources/src/BoundaryConditions/Validation/BESTEST/WeatherBESTEST.py

```diff
@@ -0,0 +1,815 @@
+# at the end it will create a folder in the current working directory
+# called simulations and inside there can be the .mat files and the
+# .json files or just the .json files
+# Creates folders in "Temp" directory
+# Copies library from local-git-hub repository
+# executes simulations and prints results in current working directory
+# ettore.zanetti@polimi.it                                 2020-03-11
+#####################################################################
+import json
+import os
+import shutil
+import numpy as np 
+import copy
+from datetime import date
+# Make code Verbose
+CodeVerbose = True
+# check if it just implements post-process (from .mat files to Json files)
+POST_PROCESS_ONLY = False
+# Erase old .mat files
+CLEAN_MAT = True
+# Erse anything but the Json file results in the ResultJson folder and .matfiles
+DelEvr = False
+#Get IBPSA library from gitHub
+FROM_GIT_HUB = False
+# Modelica IBPSA Library working branch
+#BRANCH = 'issue1314_BESTEST_weather'
+BRANCH ='master'
+#Path to local library copy
+LIBPATH = "C:\\Users\\ettor\\Documents\\Repositories\\MODELICA\\TRY\\modelica-ibpsa\\IBPSA"
+# simulator, Dymola
+TOOL = 'dymola'
+
+# List of cases and result cases
+PACKAGES = 'IBPSA.BoundaryConditions.BESTEST.Validation'
+
+CASES = ['WD100','WD200','WD300','WD400','WD500','WD600']
+
+
+reVals= ['Azi000Til00.H', 'Azi000Til00.HPer', 'Azi000Til00.HDir.H', 'Azi000Til00.HDiffIso.H', \
+ 'Azi000Til00.HDiffPer.H', 'Azi000Til90.H', 'Azi000Til90.HPer', 'Azi000Til90.HDir.H', \
+ 'Azi000Til90.HDiffIso.H', 'Azi000Til90.HDiffPer.H', 'Azi270Til90.H', 'Azi270Til90.HPer',\
+ 'Azi270Til90.HDir.H', 'Azi270Til90.HDiffIso.H', 'Azi270Til90.HDiffPer.H', \
+ 'Azi180Til90.H', 'Azi180Til90.HPer', 'Azi180Til90.HDir.H', 'Azi180Til90.HDiffIso.H',\
+ 'Azi180Til90.HDiffPer.H', 'Azi090Til90.H', 'Azi090Til90.HPer', 'Azi090Til90.HDir.H',\
+ 'Azi090Til90.HDiffIso.H', 'Azi090Til90.HDiffPer.H', 'Azi315Til90.H', 'Azi315Til90.HPer',\
+ 'Azi315Til90.HDir.H', 'Azi315Til90.HDiffIso.H', 'Azi315Til90.HDiffPer.H', 'Azi045Til90.H',\
+ 'Azi045Til90.HDir.H', 'Azi045Til90.HDiffIso.H', 'Azi045Til90.HDiffPer.H', 'Azi270Til30.H', \
+ 'Azi270Til30.HDir.H', 'Azi270Til30.HDiffIso.H', 'Azi270Til30.HDiffPer.H', 'Azi000Til30.H', \
+ 'Azi000Til30.HDir.H', 'Azi000Til30.HDiffIso.H', 'Azi000Til30.HDiffPer.H', 'Azi000Til30.HPer',\
+ 'Azi090Til30.H', 'Azi090Til30.HPer', 'Azi090Til30.HDir.H', 'Azi090Til30.HDiffPer.H', \
+ 'toDryAir.XiDry', 'weaBus.pAtm', 'weaBus.TDryBul', 'weaBus.relHum', 'weaBus.TBlaSky', \
+ 'weaBus.TDewPoi', 'weaBus.TWetBul', 'weaBus.nOpa', 'weaBus.nTot', 'weaBus.winDir', \
+ 'weaBus.winSpe', 'weaBusDew.TBlaSky','Azi270Til30.HPer','Azi045Til90.HPer','Azi090Til30.HDiffIso.H']
+
+
+
+#Software specifications 
+modeler_organization = 'LBNL'
+modeler_organization_for_tables_and_charts ='LBNL'
+program_name_and_version = 'IBPSA v4.0.0dev commit: 44c8cb5446f16e6f241494fb47f7d61418248f69, Buildings v7.0.0dev 56a796c3f2b93d86e5ea8c9aa3084457b160a478'
```

Comment:
I included it. However, I could not find a way to get repository information without cloning it, and since I am not using the Buildings library I removed it from the program_name_and_version tag. However, I made a few changes so that it should be easier to modify the script for the Buildings library